modules/protocol/inspircd.c

```diff
@@ -1347,11 +1347,18 @@ static void m_encap(sourceinfo_t *si, int parc, char *parv[])
 		if (parc < 6)
 			return;
 
+		(void) memset(&smsg, 0x00, sizeof smsg);
```

Comment:
A quick search suggests we tend to avoid casting the `memset()` return to `void`. As for `0x00`, it looks like most existing code varies between `0` and `'\0'` (though pbkdf2v2 has `0x00` as well – perhaps unsurprisingly, given that as I recall, you wrote that module); I don't really mind, though.